fairscale/nn/__init__.py

```diff
@@ -4,20 +4,7 @@
 # LICENSE file in the root directory of this source tree.
 
 from .data_parallel import FullyShardedDataParallel, ShardedDataParallel
-from .misc import FlattenParamsWrapper
+from .misc import FlattenParamsWrapper, checkpoint_wrapper
 from .moe import MOELayer, Top2Gate
 from .pipe import Pipe, PipeRPCWrapper
-from .wrap import auto_wrap, enable_wrap, wrap
-
-__all__ = [
```

Comment:
I remove those since it seems unnecessary when we basically export everything imported.

Reply:
This is not accurate. __all__ affects the behavior of 'import *'. I would recommend restoring it.

https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

Reply:
Thanks for taking a look! I can do that. OTOH, We don't maintain this consistently though. Also, is "import *" a good thing to have?

Reply:
Not sure. It might be nice to properly support `import *` but I don't have strong opinions:

https://twitter.com/jeremyphoward/status/1233062899406405636

Reply:
We should really document what should be in `__all__` and what we should be importing via `__init__.py`. I've been trying to follow PyTorch conventions but we should make those conventions explicit in the CONTRIBUTING guide.

Reply:
Good points. I will restore it back here. Let me open an issue for further discussion? I think it is slightly better not to support import *. I will provide more details in the issue.

Reply:
put it back and cleaned it up for now.